fix mdbook test with ```ignore/text/sh/console

guswynn opened the pull request:
I revitalized the dead PR #7832

sorry about the delay

testing is just running `mdbook test` which now passes! Let me know if you want me to add mdbook test to ci as well!

rust-highfive commented:
Thanks for the pull request, and welcome! The Rust team is excited to review your changes, and you should hear from @ehuss (or someone else) soon.

If any changes to this PR are deemed necessary, please add them as extra commits. This ensures that the reviewer can see what has changed since they last reviewed the code. Due to the way GitHub handles out-of-date commits, this should also make it reasonably obvious what issues have or haven't been addressed. Large or tricky changes may require several passes of review and changes.

Please see [the contribution instructions](https://github.com/rust-lang/cargo/blob/master/CONTRIBUTING.md) for more information.

guswynn commented:
I rebased, it should work now. A weird behavior of mdbook is that 

```
text
     command
more text
```

test command, I moved these to ```sh blocks in the `commands/` dir, which slightly changes the look, let me know if there is a better solution

ehuss commented:
Thanks!

@bors r+

bors commented:
:pushpin: Commit a6e322bc983629a486390cd7beb1af2f145dcffa has been approved by `ehuss`

<!-- @bors r=ehuss a6e322bc983629a486390cd7beb1af2f145dcffa -->
<!-- homu: {"type":"Approved","sha":"a6e322bc983629a486390cd7beb1af2f145dcffa","approver":"ehuss"} -->

bors commented:
:hourglass: Testing commit a6e322bc983629a486390cd7beb1af2f145dcffa with merge e1d3e13637b643881ce0327a69922c91348d72c2...
<!-- homu: {"type":"BuildStarted","head_sha":"a6e322bc983629a486390cd7beb1af2f145dcffa","merge_sha":"e1d3e13637b643881ce0327a69922c91348d72c2"} -->

bors commented:
:sunny: Test successful - [checks-actions](https://github.com/rust-lang/cargo/runs/1072333409)
Approved by: ehuss
Pushing e1d3e13637b643881ce0327a69922c91348d72c2 to master...
<!-- homu: {"type":"BuildCompleted","approved_by":"ehuss","base_ref":"master","builders":{"checks-actions":"https://github.com/rust-lang/cargo/runs/1072333409"},"merge_sha":"e1d3e13637b643881ce0327a69922c91348d72c2"} -->

bors commented:
:sunny: Test successful - [checks-actions](https://github.com/rust-lang/cargo/runs/1072333409)
Approved by: ehuss
Pushing e1d3e13637b643881ce0327a69922c91348d72c2 to master...
<!-- homu: {"type":"BuildCompleted","approved_by":"ehuss","base_ref":"master","builders":{"checks-actions":"https://github.com/rust-lang/cargo/runs/1072333409"},"merge_sha":"e1d3e13637b643881ce0327a69922c91348d72c2"} -->
